app/pods/publications/publication/proofs/documents/controller.js

```diff
@@ -178,6 +178,14 @@ export default class PublicationsPublicationProofsDocumentsController extends Co
     this.send('refresh');
   }
 
+  @action
+  async saveProofInfo(proofInfo){
+    this.publicationSubcase.dueDate = proofInfo.dueDate;
+    this.publicationSubcase.proofPrintCorrector = proofInfo.corrector;
+
+    await this.publicationSubcase.save();
+  }
```

Comment:
Te bekijken met @mdv-professional: in #1173 wordt de `save` volledig afgehandeld in de component en wordt dit niet doorgestuurd tot in de controller. We kunnen daar best consistent in zijn, denk ik.

Door de manier waarop de designs nu opgevat zijn met verschillende panels met elk een eigen save-button ben ik geneigd om de `save` in de component te doen. En met een `didSave`-hook te werken als de controller daarvan op de hoogte moet zijn. Wat denk jij @MikiDi ?

Reply:
Akkoord, op voorwaarde dat we er ons van bewust zijn dat we van het gewoonlijke patroon afwijken, en bijgevolg idd ook de benoeming van actions anders moet. `didSave` is prima voor een actie die zonet gebeurd is :ok_hand: